analytics/policy/services/business_licence_client.py

```diff
@@ -67,7 +67,7 @@ def get_licence_status(self, licence_number: str) -> list[str]:
         )
         response_data = self._make_request(fields)
 
-        return self._process_licence_status_results(response_data)
+        return self._process_licence_status_results(response_data), response_data["total_count"]
```

Comment:
this method should only return statuses, so if you want to check number of statuses, please use length of returned list